docs/troubleshooting/firebase-issues/what_to_do if_firestore_indexes_are_not_deployed.md

```diff
@@ -0,0 +1,52 @@
+---
+keywords: ['firebase', 'deployed', 'firestore']
+slug: /what-to-do-if-firestore-indexes-are-not-deployed
+title: What to Do If Firestore Indexes Are Not Deployed
+---
+# What to Do If Firestore Indexes Are Not Deployed
```

Comment:
Mentioned this many times we dont have start titles with a question. please change the title.